pkg/avancement/service_manager_test.go

```diff
@@ -203,6 +203,11 @@ func TestGenerateBranchWithSuccess(t *testing.T) {
 	GenerateBranchWithSuccess(t, repo)
 }
 
+func TestGenerateBranchForLocalSource(t *testing.T) {
+	source := NewLocal("/path/to/topLevel")
+	GenerateBranchForLocalWithSuccess(t, source)
```

Comment:
This could be unexported?
(same with `GenerateBranchWithSuccess`)

Reply:
Fixed